### PR TITLE
Add curses.COLORS and curses.COLOR_PAIRS

### PR DESCRIPTION
**Pull request by asottile**
taking precedence from #1596

### CODE REVIEW

File: stdlib/2and3/curses/__init__.pyi [hunk 1]
```diff
@@ -7,7 +7,7 @@ _T = TypeVar('_T')
 LINES: int
 COLS: int
 
-# available after calling `curses.init_color()`
+# available after calling `curses.start_color()`
```
Comment (JelleZijlstra) [inline stdlib/2and3/curses/__init__.pyi]: you actually need to call both start_color() and then init_color()

Comment (asottile) [reply stdlib/2and3/curses/__init__.pyi]: hmmmmmmmm https://github.com/python/cpython/blob/dc3f99fa77f415077c20a9c2b3e953e5cd894076/Lib/curses/__init__.py#L42-L49

Comment (JelleZijlstra) [reply stdlib/2and3/curses/__init__.pyi]: You're right, I guess I didn't check between calling start_color() and init_color() :)

Comment (asottile) [reply stdlib/2and3/curses/__init__.pyi]:
```
$ python3 -c 'import curses; curses.initscr(); curses.start_color(); curses.COLORS; curses.endwin()'
$
```

works for me

**Pull request merged**